### PR TITLE
Fix/normalize_xsdstring

### PR DESCRIPTION
**Pull request by JoshuaFortriede**
The previous PR did not include all xsdstring fixes. I added the global flag "g" to the end of the sed command to ensure all hasDbXref annotations within a single line are updated, not just the first one. This ensures that the command does not need to be run multiple times.

### CODE REVIEW

**Comment by matentzn** [comment]
@rays22 can you review this? I am not sure what the /g does, you are better at that kind of stuff.

File: docs/Fixing_xsdstring_diffs.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ When you make edits, sometimes there will be large amounts of unintended differe
 
   If you do not have docker installed: ```make normalise_xsd_string```
 
-  If "make" is not installed, on MAC: ``` sed -i '' "s/Annotation[(]\(oboInOwl[:]hasDbXref [\"][^\"]*[\"]\)[)]/Annotation(\1^^xsd:string)/" cl-edit.owl ```
+  If "make" is not installed, on MAC: ``` sed -i '' "s/Annotation[(]\(oboInOwl[:]hasDbXref [\"][^\"]*[\"]\)[)]/Annotation(\1^^xsd:string)/g" cl-edit.owl ```
```
Comment (rays22) [inline docs/Fixing_xsdstring_diffs.md:16]: The `g` global flag is fine and it is an improvement.

File: src/ontology/cl.Makefile [hunk 2]
```diff
@@ -272,7 +272,7 @@ all_reports: reports/obo-diff.txt
 
 
 normalise_xsd_string: $(SRC)
-	sed -i.bak -E "s/Annotation[(](oboInOwl[:]hasDbXref [\"][^\"]*[\"])[)]/Annotation(\1^^xsd:string)/" $<
+	sed -i.bak -E "s/Annotation[(](oboInOwl[:]hasDbXref [\"][^\"]*[\"])[)]/Annotation(\1^^xsd:string)/g" $<
```
Comment (rays22) [inline src/ontology/cl.Makefile:275]: The global flag is an improvement and looks fine.

**Review by rays22 — APPROVED** [review]
The `sed /g` flag is a great improvement.

**Pull request merged**